README.md

```diff
@@ -32,6 +34,12 @@ Devs can also test the deployment build of the frontend by running the following
 $ ROPE_APP_TARGET=deploy docker compose up --build -d
 ```
 
+When developing backend code, developers may want to install the package in an editable virtual environment. That can be done as follows (to avoid unresolved imports per [this issue](https://github.com/microsoft/pylance-release/issues/3473)):
```

Comment:
Should we also mention needing to restart the language server on VSCode?

Reply:
Yeah, that's a good note to include. I'll add it.